docs/user-guide.md

```diff
@@ -98,56 +118,73 @@ In the conversation header, you can view the current settings, change settings (
 
 ### Share
 
-You can share a conversation or a folder with several conversations with other users.
+You can share a conversation or a folder with several conversations with other users. Yo can also receive a shared conversation.
 
-> When someone shared a conversation with you, you can find it in the **Shared with me** section on the left panel. **Important**: you cannot change a conversation that has been shared with you. To be able to work with it, [duplicate it](#duplicate).
+**To receive a shared conversation**:
 
-To share a conversation or a folder, click **Share** in the contextual menu and copy the link. After someone has opened your link, an arrow icon appears near its name and this conversation can be found when **Shared by me** checkbox is selected in the **Search** menu.
+When someone shared a conversation with you, you can find it in the **Shared with me** section in the left panel. 
 
-![](./img/shared.png)
+![](./img/shared-with-me.png)
+
+> **Important**: you cannot change a conversation that has been shared with you. To be able to work with it, [duplicate it](#duplicate).
+
+**To share a conversation**:
 
-When you share a folder, all folders and conversations in this folders will be shared.
+> **Note**: You can share entire folders with conversations. When you share a folder, all folders and conversations in it will be shared as well.
+
+To share a conversation or a folder, click **Share** in the contextual menu and copy the link.
 
 ![](./img/share_conv.png)
 
+After the recipient has opened your link, the arrow icon appears near its name.
```

Comment:
Actually user has to re-login or refresh the browser to see the arrow for shared chat/prompt. Do two conditions: to click the link by user2 and refresh the page by user1